ruby-push-notifications.gemspec

```diff
@@ -20,6 +16,7 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ['lib']
 
+  spec.add_dependency 'activesupport', '~> 4.2'
```

Comment:
I think it is a dependency that [builder gem](https://github.com/calonso/ruby-push-notifications/blob/master/ruby-push-notifications.gemspec#L23) brings. I had to fix it because the current version is 5.x and it is not compatible with Ruby <= 2.1.x